### PR TITLE
Exclude environment variables which are not defined

### PR DESCRIPTION
**Pull request by jsok**
Addresses #1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,8 +3,12 @@ var util = require('util');
 
 var DECLARATION = 'window.__env__ = window.__env__ || {};\n',
     TEMPLATE = function(name) {
-      var tmpl = 'window.__env__[\'%s\'] = \'%s\';\n'
-      return util.format(tmpl, name, process.env[name]);
+      if (typeof process.env[name] !== 'undefined') {
+        var tmpl = 'window.__env__[\'%s\'] = %s;\n';
```
Comment (Kagami) [inline index.js]: Should it be `var tmpl = 'window.__env__[\'%s\'] = \'%s\';\n';` instead?

**Comment by jsok** [comment]
Nice catch, thanks.

File: index.js [hunk 2]
```diff
@@ -3,8 +3,12 @@ var util = require('util');
 
 var DECLARATION = 'window.__env__ = window.__env__ || {};\n',
     TEMPLATE = function(name) {
-      var tmpl = 'window.__env__[\'%s\'] = \'%s\';\n'
-      return util.format(tmpl, name, process.env[name]);
+      if (typeof process.env[name] !== 'undefined') {
+        var tmpl = 'window.__env__[\'%s\'] = \'%s\';\n';
```
Comment (Kagami) [inline index.js:7]: Also, it will break if string contains single quotas. I thing it's better to use double quotes and `JSON.stringify`.

**Comment by Kagami** [comment]
You were too fast :)

**Pull request merged**